airflow-core/docs/core-concepts/auth-manager/index.rst

```diff
@@ -172,6 +172,39 @@ cookie named ``_token`` before redirecting to the Airflow UI. The Airflow UI wil
 .. note::
     Do not set the cookie parameter ``httponly`` to ``True``. Airflow UI needs to access the JWT token from the cookie.
 
+Refreshing JWT Token
+''''''''''''''''''''
+The refresh token endpoint is ``POST /auth/token/refresh`` with the current JWT token in the cookie or acquired from the API.
```

Comment:
Not sure I understand this sentence

Reply:
I split the sentence into multiple pieces. I beleive this is more understanable